server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+export EXTRA_PODMAN_SWITCHES=${EXTRA_PODMAN_SWITCHES:-"--pull=always"}
+export IMAGE_KIND=${IMAGE_KIND:-"fedora"}
+export IMAGE_REPO=${IMAGE_REPO:-"quay.io/pbench"}
+export IMAGE_ROLE=${IMAGE_ROLE:-"ci"}
```

Comment:
I don't think these definitions should be here:  this file doesn't invoke `jenkins/run` or anthing which depends on it, so these definitions belong in whatever file is doing that (so they don't get "hidden").  (And, they are already in `jenkins/run-server-func-tests`.)